tutorials/README.md

```diff
@@ -0,0 +1,123 @@
+# AffinityVAE Tutorial
+
+## Running AffinityVAE on the MNIST dataset
+
+### 1. Install AffinityVAE
+
+As described in the [README](../README.md), AffinityVAE can be installed using
+pip:
+
+```bash
+python -m venv env
+source env/bin/activate
+python -m pip install --upgrade pip
+python -m pip install -e .
+```
+
+### 2. Download the MNIST dataset
+
+The MNIST dataset can be downloaded from:
+https://figshare.com/articles/dataset/mnist_pkl_gz/13303457
```

Comment:
This link comes out as page unavailable which confused me for a few minutes. is it easier to have this as a `wget command`

`wget https://figshare.com/ndownloader/files/25635053`